core-web/libs/edit-content/src/lib/components/dot-create-content-dialog/dot-edit-content-dialog.component.scss

```diff
@@ -0,0 +1,97 @@
+@use "variables" as *;
+
+.edit-content-dialog {
+    height: 100%;
+    display: flex;
+    flex-direction: column;
+    min-height: 60vh;
+}
+
+// Loading state
+.edit-content-dialog__loading {
+    display: flex;
```

Comment:
We should use the Prime Flex utilities classes like this: `<div class="flex flex-column align-items-center justify-content-center text-center flex-1" >`